include/flatbuffers/flexbuffers.h

```diff
@@ -743,7 +745,7 @@ inline Reference GetRoot(const uint8_t *buffer, size_t size) {
 }
 
 inline Reference GetRoot(const std::vector<uint8_t> &buffer) {
-  return GetRoot(buffer.data(), buffer.size());
+  return GetRoot(flatbuffers::vector_data(&buffer), buffer.size());
```

Comment:
I've replaced everything with a reference though I *really* don't like it.  It simply aids to obfuscate the code at the call site.  This is something that could be resolved during a style pass.